client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "corona-tracker",
-  "version": "0.3.36",
+  "version": "0.3.37",
```

Comment:
Will need to bump to most recent version based off of master

Reply:
done